src/test/java/mate/academy/lib/InjectorTest.java

```diff
@@ -13,7 +13,7 @@
 import java.lang.reflect.Field;
 import java.util.Arrays;
 
-class InjectorTest {
+class git InjectorTest {
```

Comment:
There is a typo in the class declaration. It should be `class InjectorTest` instead of `class git InjectorTest`.